src/components/Link/Link.jsx

```diff
@@ -31,12 +32,14 @@ Link.propTypes = {
   icon: PropTypes.string,
   label: PropTypes.string.isRequired,
   style: PropTypes.shape({}),
+  target: PropTypes.string,
 }
 
 Link.defaultProps = {
   addClass: '',
   icon: '',
   style: {},
+  target: ''
```

Comment:
the default value for this attribute is `target: _self`. (ref: [docs](https://www.w3schools.com/tags/att_a_target.asp)) what behavior does an empty string perform?

Reply:
it does the exact same thing as `target='_self'`, but i will change it to self if for best practices purposes